### PR TITLE
PMM-4214 Configure pmm-agent.

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-308-2480f98
client docker - perconalab/pmm-client-fb:PR-308-2480f98
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-308-2480f98.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-308-c15cfa8
client docker - perconalab/pmm-client-fb:PR-308-c15cfa8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-308-c15cfa8.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-308-88fa26c
client docker - perconalab/pmm-client-fb:PR-308-88fa26c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-308-88fa26c.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-308-e979764
client docker - perconalab/pmm-client-fb:PR-308-e979764
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-308-e979764.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-308-9c7a236
client docker - perconalab/pmm-client-fb:PR-308-9c7a236
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-308-9c7a236.tar.gz

**Comment by AlekSi** [comment]
@BupycHuk @hors PTAL

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-308-57ca566
client docker - perconalab/pmm-client-fb:PR-308-57ca566
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-308-57ca566.tar.gz

File: build/pmm-server-docker/playbook-init.yml [hunk 1]
```diff
@@ -113,19 +130,28 @@
...
+        - /srv/logs/clickhouse-server.log
+        - /srv/logs/clickhouse-server.startup.log
         - /srv/logs/cron.log
         - /srv/logs/dashboard-upgrade.log
-        - /srv/logs/nginx.log
-        - /srv/logs/node_exporter.log
+        - /srv/logs/grafana.log
+        - /srv/logs/nginx.startup.log
+        - /srv/logs/pmm-agent.log
+        # - /srv/logs/pmm-configurator.log
```
Comment (hors) [inline build/pmm-server-docker/playbook-init.yml]: maybe we need to remove all commented lines

Comment (AlekSi) [reply build/pmm-server-docker/playbook-init.yml]: pmm-configurator / pmm-manage logs are very useful while I work on updates, etc. I added them explicitly to remove later – when pmm-configuration is removed.

**Pull request merged**